kubetest/azure.go

```diff
@@ -925,10 +927,18 @@ func (c Cluster) TestSetup() error {
 		if err := os.Setenv("K8S_AZURE_LOCATION", c.location); err != nil {
 			return err
 		}
+	} else if *testAzureFileCSIDriver == true {
+		// Set env vars required by Azure File CSI driver tests.
+		// tenantId, subscriptionId, aadClientId, and aadClientSecret will be obtained from AZURE_CREDENTIAL
+		if err := os.Setenv("resourceGroup", c.resourceGroup); err != nil {
```

Comment:
could we rename the env name for resourceGroup and location? They're usually in capital letter.